tests/Dockerfile

```diff
@@ -6,6 +6,7 @@ ARG ORG=opendatahub-io
 ARG BRANCH=master
 ARG ODS_CI_REPO=https://github.com/red-hat-data-services/ods-ci
 ARG ODS_CI_GITREF=releases/1.7.0-5
+ARG OC_CLI_URL=https://mirror.openshift.com/pub/openshift-v4/amd64/clients/ocp/latest-4.10/openshift-client-linux.tar.gz
```

Comment:
Shoud we keep this  as latest (https://mirror.openshift.com/pub/openshift-v4/amd64/clients/ocp/latest/openshift-client-linux.tar.gz), to avoid updating for future releases?

Reply:
Great idea